package.json

```diff
@@ -78,6 +82,8 @@
     "transform": {
       "\\.(ts)$": "ts-jest"
     },
+    "testEnvironment": "node",
+    "testRunner": "jest-circus/runner",
```

Comment:
By default, if the `beforeAll` hook throws an error during the test, jest will not automatically fail the test.

This config option fixes that.
https://github.com/facebook/jest/issues/2713